integration-tests/.yourbase.yml

```diff
@@ -0,0 +1,51 @@
+# NOTE first iteration, just testing `yb build`
+# Tries building 3 known-good Python FOSS projects
+# After fixing how the CLI manages workspaces (currently done using config.yml), we could simplify this a bit
+#
+build_targets:
+  - name: default
+    commands:
+      # TODO, add submodule awareness to the CLI
```

Comment:
This seems like an issue with our build environment, right? I'm not sure why our build tool should know about the version control system being used.

Definitely file a ticket for that, if one doesn't already exist.

Reply:
:heavy_check_mark: https://app.clubhouse.io/yourbaseio/story/1746/the-cli-tool-should-have-git-sub-module-awareness